MicroProfile-JWT-Auth/tck-arquillian-extension/pom.xml

```diff
@@ -57,6 +57,42 @@
         <!-- JSON Dependencies -->
         <json.version>1.1.2</json.version>
     </properties>
+    
+    <profiles>
+       <!-- This profile will exclude the TokenParser class and use the one in Payara Server instead. For versions 5.193+ -->
+        <profile>
+            <id>payara5</id>
```

Comment:
`payara5` is not a suitable name for this profile as activated based on regex only for 5.193+ version.